dvc/lock.py

```diff
@@ -15,6 +15,12 @@
 
 DEFAULT_TIMEOUT = 5
 
+ERROR_MESSAGE = (
+    "cannot perform the command since other dvc process is running for this "
+    "repo. If you are sure this is not the case, you might remove .dvc/lock "
+    "to fix it."
```

Comment:
> Updating https://github.com/iterative/dvc/pull/2765#pullrequestreview-316566729

My suggestion:

FAILED_TO_LOCK_MESSAGE="Cannot perform the command. Another \`dvc\` process may be running on this project. If this is not the case, manually remove .dvc/lock and try again."

cc @shcheklein